### DISCUSSION TITLE
Question about dpdk in Unikraft

### DISCUSSION THREAD

**Discussion by Zack-Will** [unikraft/unikraft#621] [Q&A]
Recently I’m learning about the Unikraft and try to transplant dpdk as a Unikraft lib. When I was searching for it, I noticed that your team have already done this work in 2019. Will this part of the code be open sourced in the unikraft project? If not, may I get the glue code for research?

**Comment by razvand** [comment]
@Zack-Will, since we are already (viciously) using [the Unikraft Discord server](https://bit.ly/UnikraftDiscord) for discussions, meetings, brainstorming and jokes (mostly bad, we do what we can), we will close the Discussions tab. We will do this on Friday, March 31, 2023.

Please move any discussions or questions on [the Unikraft Discord server](https://bit.ly/UnikraftDiscord). See you all there! :handshake:
